dashboard/db/schema.rb

```diff
@@ -457,6 +457,16 @@
     t.index ["name"], name: "index_courses_on_name"
   end
 
+  create_table "data_docs", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci", force: :cascade do |t|
```

Comment:
unfortunately I am seeing some diffs when I apply this migration locally:
```
Dave-MBP:~/src/cdo/dashboard (staging)$ UTF8=1 RAILS_ENV=test bundle exec rake db:reset db:test:prepare

Dave-MBP:~/src/cdo/dashboard (staging)$ git checkout add-data-guide-model

Dave-MBP:~/src/cdo/dashboard (add-data-guide-model)$ RAILS_ENV=test bundle exec rake db:migrate                  
== 20220525181706 CreateDataDocs: migrating ===================================
-- create_table(:data_docs)
   -> 0.0277s
== 20220525181706 CreateDataDocs: migrated (0.0278s) ==========================

Dave-MBP:~/src/cdo/dashboard (add-data-guide-model *)$ git diff
diff --git a/dashboard/db/schema.rb b/dashboard/db/schema.rb
index 6f51582e9e4..5a43dc3764d 100644
--- a/dashboard/db/schema.rb
+++ b/dashboard/db/schema.rb
@@ -457,7 +457,7 @@ ActiveRecord::Schema.define(version: 2022_05_25_181706) do
     t.index ["name"], name: "index_courses_on_name"
   end
 
-  create_table "data_docs", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COLLATE=utf8mb4_unicode_ci", force: :cascade do |t|
+  create_table "data_docs", options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci", force: :cascade do |t|
     t.string "key", null: false
     t.string "name"
     t.text "content"
```

what I would suggest is that you run the same steps that I have run here, and then commit the result to this PR. it should result in the "mb4" parts of this line going away.

apologies if you have heard contrary advice from others... I forget where we are at in the debate on this, so if you believe this will all get sorted out on staging then that could also be ok -- but it's worth noting that as long as one of your local databases has utf8mb4 as the default, you're likely to keep running into problems like this.

Reply:
Lol I thought I took care of that problem ... will try again. Thanks!

Reply:
the above steps should get your local test database into a good state. if you are on board with what I'm suggesting, then you'll want to also get your local development db into a good state (step 11 in https://github.com/code-dot-org/code-dot-org/blob/2f70349a4548981b573392200b8960a92d75b6eb/SETUP.md#overview ). 

then you would also need to run the migration backward and forward again on your development DB to get it to pick up the utf8 settings.